core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -247,6 +247,14 @@
   exploration-editor-page .navbar-icons li:hover {
     background-color: #4eb0c0;
   }
+  .oppia-alert-warning {
```

Comment:
Missing the prefix here. Please ensure prefixes are added everywhere as mentioned in above comments.